libs/coin-evm/src/getTransactionStatus.ts

```diff
@@ -104,12 +108,25 @@ export const validateAmount = (
 export const validateGas = (
   account: Account,
   tx: EvmTransaction,
-  gasLimit: BigNumber,
   estimatedFees: BigNumber,
+  gasLimit: BigNumber,
+  customGasLimit?: BigNumber,
 ): Array<ValidationIssues> => {
   const errors: ValidationIssues = {};
   const warnings: ValidationIssues = {};
 
+  // Gas Limit
+  if (gasLimit.isZero()) {
+    errors.gasLimit = new FeeNotLoaded(); // "Could not load fee rates. Please set manual fees"
+  } else if (gasLimit.isLessThan(21000)) {
```

Comment:
could you make this a constant ?

Reply:
Already exists in transaction.ts if you need want it

Reply:
Already exists in transaction.ts if you need want it